mbq/client/contrib/permissions.py

```diff
@@ -64,6 +63,11 @@ def fetch_permissions_for_location(
     def fetch_all_permissions(self, person_id: str) -> FetchedPermissionsDoc:
         ...
 
+    def fetch_orgs_for_permission(
+        self, person_id: str, scope: str
+    ) -> Iterable[Union[UUIDType, str]]:
```

Comment:
str is already in the type

Reply:
also should we not make this always return str? we have full control over it so we can be more opinionated. the UUIDType is better for inputs